nickytonline/iamdeveloper
# feat: added talk pages

- feat: added talk pages
- chore: added talks layout
- chore: refactored talks page

## What type of PR is this? (check all applicable)


## Description

Adds an index page that lists all talks and individual pages for each talk.

## Related Tickets & Documents



- Related Issue #
- Closes #9

## QA Instructions, Screenshots, Recordings

1. Navigate to https://deploy-preview-8--iamdeveloper-dot-com-beta.netlify.app/talks/
2. A list of talks appears.
<img width="950" alt="CleanShot 2023-03-11 at 00 08 22@2x" src="https://user-images.githubusercontent.com/833231/224466144-e3912068-1c6d-4759-92ee-70ab9fa9dade.png">

4. Click on one of the links for a talk.
5. The individual talks page loads up.

<img width="994" alt="CleanShot 2023-03-11 at 00 08 49@2x" src="https://user-images.githubusercontent.com/833231/224466166-ce500215-7200-475e-82a4-9211404e7510.png">

### UI accessibility concerns?

N/A

## Checklist

- [x] Open a [bug/issue](https://github.com/nickytonline/iamdeveloper/issues/new/choose) before writing your code 🧑‍💻. This ensures we
      can discuss the changes and get feedback from everyone that should be involved. If you\`re fixing a typo or
      something that\`s on fire 🔥 (e.g. incident related), you can skip this step.
      guide and passes our tests.

### Added/updated tests?

- [x] No, and this is why: testing is not in place yet.

## [optional] What gif best describes this PR or how it makes you feel?
